### ISSUE TITLE
JS optimizer still breaking site

### ISSUE THREAD

**Issue by revendo** [litespeedtech/lscache_wp#22]
The Bug which is supposed to be fixed in Version 1.7.2 still exists. When turning on JS Minify and JS Combine, a normal Wordpress setup breaks because of JS issues. 
<img width="905" alt="bildschirmfoto 2018-01-07 um 20 17 47" src="https://user-images.githubusercontent.com/13151469/34653137-dcfcd43c-f3e7-11e7-92ef-6ea72f10358d.png">

[BUGFIX] Fixed JS optimizer breaking certain plugins JS.

**Comment by hailite** [comment]
If you turned off `Remove Comment` but still can see this issue, that means you have a optimized file cache which is under `/tmp/minify__*`. You need to run `rm /tmp/minify__*` for now, following by a Purge All. It should work. 

We will improve this JS optimizer to clear that file cache automatically later.

**Comment by revendo** [comment]
I `ran rm /tmp/minify__*`, Purged All (including Cloudflare) and reenabled `JS Minify` and `JS Combine`. Same issues. Do I need to turn off `Remove Comment`? That feature doesn't cause ani issue. But maybe together with the other JS related settings?

**Comment by hailite** [comment]
Yes please try to turn it off. Thats the only changes in these releases to JS optm.

**Comment by revendo** [comment]
I tried to turn it off (followed by purge all). Same result.

**Comment by hailite** [comment]
Do you have any cdn cache? Did you remove tmp files?

**Comment by revendo** [comment]
As mentioned, I purged all including Cloudflare. On the test server, which isn't running any CDN, it also doesn't work.

**Comment by hailite** [comment]
What if you turn off JS Combine?
